index.html

```diff
@@ -60,144 +66,126 @@ <h1>Hey there,<br>I´m cesar</h1>
                 <a href="#mediumlink"><img src="imgs/Mediumimg.png" alt="mediumimg"></a>
             </li>
         </ul>
+    </div>
     </section>
     <section class="proyects">
-        <h3 class="fontstitles">My Recent Works</h3>
+        <div class="tittleworksproyects">
+            <h3 class="fontstitles">My Recent Works</h3>
+            <div class="line">
+            </div>
+        </div>
+    <div class="works">
         <div class="imgprincipal">
-            <img src="imgs/ImgPlaceholder.jpg" alt="placeholder">
         </div>
         <div class="grid">
             <div>
-                <h4 class="letter">Multi-post Stories</h4>
+                <h3 class="letter">Multi-post Stories</h3>
             </div>
-            <div class="texthead">
                 <p class="intro1">A daily selection of privately personalized reads;
                     no accounts or sign-ups required. has been the industry's
                     standard dummy text ever since the 1500s,when an unknown printer
                     took a standard dummy text.</p>
-            </div>
-            <ul>
+            <ul class="programs">
                 <li class="css2">css</li>
                 <li class="html2">html</li>
                 <li class="boostrap2">boostrap</li>
                 <li class="ruby2">ruby</li>
             </ul>
-            <div><button type="submit" class="btnseeproyect">See project</button></div>
-        </div>
-        <div class="grid2">
-            <div>
```

Comment:
- [ ] The section marked red is the state of that project card on `hover`, and `focus `while the ones marked blue in the screenshot below is the normal state. The first layer disappears and the button appears. in other words, when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears to the bottom of the image and takes the whole image width. _**You can go ahead and include transition if you want**_
 👉  Kindly implement this behavior using CSS `:hover`  & `:focus` psuedoclass. 

![image](https://user-images.githubusercontent.com/84156714/186789195-cd7113db-0193-47c7-a232-7e6a529f9c80.png)

Reply:
hi, im going to implimentate CSS :hover & :focus the other thing that apears and desapear, i dont know how do that

Reply:
i did the CSS:hover and :focus

Reply:
Kindly, I recommend you follow the advice of the previous reviewer.
He tells you that the images must change when an event is performed on them, so I leave this same work that I did a few months ago so you can see how it should work (do not take into account the animations)
https://mgmediaweb.github.io/portfolio/

Reply:
but he said that You can go ahead and include transition if you want

Reply:
it's required or it's optional ??

Reply:
The previous reviewer (@bobb-Rob ) is asking you to make the mouse over effect on each project, I showed you my page so you can see how it should work, these changes are not OPTIONAL, you must make them. U can do it!!!!